composer.json

```diff
@@ -4,21 +4,14 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
-        "dpc-sdp/tide_core": "^1.0",
+        "dpc-sdp/tide_core": "dev-feature/SDPA-1213_metatag_upgrade as 1.0",
         "drupal/token_conditions": "dev-1.x",
-        "drupal/webform": "5.0.0-rc18"
+        "drupal/webform": "^5.2"
     },
     "suggest": {
-        "dpc-sdp/tide_api:^1.0": "Allows to use Drupal in headless mode",
-        "dpc-sdp/tide_landing_page:^1.0": "Landing Page content type and related configuration for Tide Drupal 8 distribution",
-        "dpc-sdp/tide_page:^1.0": "Page content type and related configuration for Tide Drupal 8 distribution"
-    },
-    "extra": {
-        "patches": {
-            "drupal/webform": {
-                "Error processing Webform Tokens, sending leaked metadata - https://www.drupal.org/project/webform/issues/3010727#comment-12838354": "https://www.drupal.org/files/issues/2018-11-01/3010727-4.patch"
```

Comment:
@sonnykt Did you tested submit a webform like "Rate this page because there is a not issue related to webform with handlers that use Token, not related to JSON API. 

IMHO this patch shouldn't be removed, also there is a Patch that did in this PR https://github.com/dpc-sdp/content-vic-gov-au/pull/425 ticket https://digital-engagement.atlassian.net/browse/SDPA-992 for Webform 5.1 maybe works for 5.2